### DISCUSSION TITLE
TanStack Router support?

### DISCUSSION THREAD

**Discussion by aconrad** [grafana/faro-web-sdk#756] [Ideas]
Are there any plans to support a [TanStack Router](https://tanstack.com/router) integration?

**Comment by codecapitano** [comment]
Hey @aconrad we don't have any plans yet due to conflicting priorities but are very open for contributions.
Can you add it as a feature request to the faro issues?

**Comment by aconrad** [comment]
Thanks @codecapitano, I created https://github.com/grafana/faro-web-sdk/issues/777.
